### DISCUSSION TITLE
Struggling to add a range of interfaces....

### DISCUSSION THREAD

**Discussion by jethro66** [netbox-community/netbox#15013] [General]
I'm using the text "FortyGig1/0/[0-32]" in the name field when adding Interfaces to a device type, under "Add Components".  I then press "create" at the bottom of the screen.......but receive a red banner message popup stating "Name  This field is required".  If I drop the square brackets from my text in the Name" field, and type in a single number representing a single interface without modifying any other fields......the interface is created....no problems

Maybe I'm tired, and I'm missing something really obvious, because I was able to create bulk interfaces using square brackets earlier today...with no problems :-(

**Comment by candlerb** [comment]
Netbox version? What exact URL path are you seeing in the browser URL bar?

It works for me. Netbox 3.7.1, URL `/dcim/interface-templates/add/?device_type=5&return_url=/dcim/device-types/5/interfaces/` with title "Add a new interface template". I entered exactly the string you gave, and QSFP+ for the interface type, nothing else.

**Reply by jethro66** [reply]
I'm running v3.2.5

https://netbox.bla_bla.bla.au/netbox/dcim/interface-templates/add/?device_type=39&return_url=/netbox/dcim/device-types/39/interfaces/

The page title is "Add a new interface template"

Oh...FYI....after just typing this....I tried again and left the "Label" field blank and it worked perfectly !!
I then tried again and put a plain text label in, ie  "QSFP+ Port".... it failed like before
I tried a 3rd time but this time I put "QSFP+ Port [1-32]" in the field....and it worked perfectly !!!

I then read the text under the "Label" Field, ie

"Alphanumeric ranges are supported. (Must match the number of names being created)"

Makes sense now !!

Thanks candlerb
